pcfuncs/funclib/models.py

```diff
@@ -104,3 +110,137 @@ def from_query_params(cls, render_params: str) -> "RenderOptions":
                 result[k] = v
 
         return RenderOptions(**result)
+
+
+@attr.s
+class RIOImage(ImageData):
```

Comment:
This class extent rio-tiler ImageData class to add methods that are either not in rio-tiler 3.1 or needed for the specific use case